src/sentry/static/sentry/app/views/eventsV2/landing.tsx

```diff
@@ -281,12 +290,18 @@ class DiscoverLanding extends AsyncComponent<Props, State> {
     }
 
     return (
-      <SentryDocumentTitle title={t('Discover')} objSlug={organization.slug}>
-        <React.Fragment>
-          <GlobalSelectionHeader organization={organization} />
-          <NoProjectMessage organization={organization}>{body}</NoProjectMessage>
-        </React.Fragment>
-      </SentryDocumentTitle>
+      <Feature
+        organization={organization}
+        features={['discover-query']}
+        renderDisabled={this.renderNoAccess}
+      >
+        <SentryDocumentTitle title={t('Discover')} objSlug={organization.slug}>
+          <React.Fragment>
```

Comment:
Do you need the fragment?

Reply:
Yea, SentryDocumentTitle gets cranky if its gets a list of children.